Open Judge System/Web/OJS.Web/Scripts/Administration/Contests/contests-index.js

```diff
@@ -1,13 +1,25 @@
-﻿function getTypeName(type) {
+﻿/* exported getTypeName */
```

Comment:
The exported comments are used to indicated which functions are public, for the specific file, so the linter can mark them as "exported" (ES5 has no way to "export" stuff).

See this for more info: 
http://eslint.org/docs/rules/no-unused-vars#exported